src/views/home/components/bridge-form/bridge-form.view.tsx

```diff
@@ -62,13 +61,37 @@ const BridgeForm: FC<BridgeFormProps> = ({ account, formData, resetForm, onSubmi
   const [estimatedFee, setEstimatedFee] = useState<AsyncTask<BigNumber, string>>({
     status: "pending",
   });
+  const [chainList, setChainList] = useState<Chain[]>();
+  const [tokenList, setTokenList] = useState<TokenWithBalance[]>();
+  const [tokensWithBalance, setTokensWithBalance] = useState<TokenWithBalance[]>();
```

Comment:
I like the classic names `chainList` and `tokenList` since these arrays represent the tokens that are "listed". We could rename them as `listedChains`/`listedTokens` or `visibleChains`/`visibleTokens` but I prefer their current names.

I don't like too much the prefix "filtered" because they don't "have to" be filtered and also, I like the consistency in the naming for both `chainList` and `tokenList`. Renaming `tokenList` as `filteredTokenList` would break that consistency.

I have though renamed `tokensWithBalance` as just `tokens`. I think it's clear that this is the "canonical list" from which we filter to create the `tokenList`.

What do you think?

Reply:
This is done